packages/controller/main.js

```diff
@@ -5679,6 +5669,36 @@ function init(compactGroupId) {
     process.on('unhandledRejection', exceptionHandler);
 }
 
+/**
+ * Parses out the rebuild path from an error log
+ *
+ * @param {string} text - log text
+ * @return {string | undefined}
+ * @private
+ */
+function _determineRebuildPathFromLog(text) {
+    // extract rebuild path - it is always between the only two single quotes
+    const matches = text.match(/'.+'/g);
```

Comment:
Why not directly use /'(.+)'/g) ... then you directl have the value in matches[2] ?! YOu do dnot need to do the string replace then

Reply:
`/'(.+)'/g` actually returns the same

Reply:
with named capture group it would be a one liner but imo harder to read and more extensive checks

```
    const matches = text.match(/'(?<rebuildPath>.+)'/);
```

thus i would keep it

Reply:
ah edit: okay with your approach correct in matches[**1**] is correct but I think forcing one match is nice too